x-pack/plugins/ml/server/lib/alerts/jobs_health_service.ts

```diff
@@ -148,6 +199,40 @@ export function jobsHealthServiceProvider(
         }
       }
 
+      if (config.mml.enabled) {
+        const response = await this.getMmlReport(jobIds);
+        if (response && response.length > 0) {
+          const hardLimitJobsCount = response.reduce((acc, curr) => {
+            return acc + (curr.memory_status === 'hard_limit' ? 1 : 0);
+          }, 0);
+
+          results.push({
+            name: HEALTH_CHECK_NAMES.mml,
+            context: {
+              results: response,
+              message:
+                hardLimitJobsCount > 0
+                  ? i18n.translate(
+                      'xpack.ml.alertTypes.jobsHealthAlertingRule.mmlHardLimitMessage',
+                      {
+                        defaultMessage:
+                          '{jobsCount, plural, one {# job} other {# jobs}} reached the hard model memory limit',
```

Comment:
```suggestion
                          '{jobsCount, plural, one {# job} other {# jobs}} reached the hard model memory limit.',
```

Reply:
Updated in b73d27b7abebb21dd393529cca351d0af5eb43bc